### ISSUE TITLE
Test a Google Meet Livestream

### ISSUE THREAD

**Issue by oindrillac** [operate-first/operate-first.github.io-old#208]
Test livestream planned for https://github.com/operate-first/SRE/issues/104 and record any steps or tips

**Comment by oindrillac** [comment]
I was able to test out the Livestream on google meet and record the meeting.

**Details:**
There would be 2 Meeting Invites - 
1. one for the organizers, who would be on the call and be sharing their screens. It will have the option to join the meeting itself or join the livestream
![image](https://user-images.githubusercontent.com/32435206/109879754-00ebb600-7c44-11eb-87f0-c911cedc8e3e.png)

2. another one for the view only attendees with an option to only join the Livestream.
![image](https://user-images.githubusercontent.com/32435206/109879792-0c3ee180-7c44-11eb-99a9-4e6a5077214d.png)

**On the Viewer End**
Viewers see meeting partcipants video and the presented screen. Here's the recording of a Test Google Meet Livestream: https://drive.google.com/file/d/1vJlfBU-BoJUXmqpog4ykwyYCx5UZea8P/view?usp=sharing. This recorded stream can be later uploaded to YT.

Only redhat.com accounts can view the Livestream for now. For external organizational access we need Administrative [permissions to enable that](https://support.google.com/a/answer/9055446).

**How to Setup**
Detailed Steps here: https://support.google.com/meet/answer/9308630?co=GENIE.Platform%3DDesktop&hl=en&authuser=1&oco=0

tldr:
1. Set up a normal google meet call with the participants of the migration and after adding the Google Meet conferencing link to the meeting, Next to `Join with Google Meet`, click `Down arrow` and then `Add live stream`.
2. Once the meeting is created, Click the live stream event you created and then `Edit`, at the top, click `More actions` and then `Create view-only event`. This is the second view-only event where we intend to invite the viewers.


cc: @durandom @HumairAK @tumido @4n4nd

**Comment by tumido** [comment]
Hey! @oindrillac this is very nice! :+1: Thank you! Can we maybe convert your guide into a doc in the repo? I think we will be using this more and more! I love the simplicity of this. :smile: :1st_place_medal:

**Comment by oindrillac** [comment]
@tumido sure, thanks! i could add a small document in the repo :+1:

**Comment by oindrillac** [comment]
operate-first/support#848
